Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions.yml

```diff
@@ -0,0 +1,165 @@
+category: Utilities
+commonfields:
+  id: GoogleCloudFunctions
+  version: -1
+configuration:
+- display: Service Account Private Key file contents (JSON)
+  name: credentials_json
+  required: true
+  type: 0
+- display: Default Project ID
+  name: project_id
+  required: false
+  type: 0
+- display: Default Region
+  name: region
+  required: false
+  type: 0
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
```

Comment:
this is not used in the code

Reply:
going to implement this with @yaakovi tomorrow morning.